src/title.js

```diff
@@ -3,8 +3,12 @@ import slidesContent from "training-material/Slides/slides.json";
 const convertTextToHtml = document.createElement("div");
 convertTextToHtml.innerHTML = slidesContent[0];
 
-const trainingTitle = convertTextToHtml.querySelector(
+const trainingTitleElement = convertTextToHtml.querySelector(
   "section h1:first-of-type"
-).textContent;
+);
+
+const trainingTitle =
+  (trainingTitleElement && trainingTitleElement.textContent) ||
+  "{Training Title}";
```

Comment:
Le résultat pourrait peut-être plus explicite sur le problème et comment le corriger, du genre :

```suggestion
  "TITLE NOT FOUND (please make sure you have a first level title (e.g. '# My title') in the first file referenced in 'Slides/slides.json'";
```

Reply:
What about `TITLE NOT FOUND (see console)` and detail the rest in a warning in the console?

Reply:
Excellente idée !

Reply:
Cependant il faut qu'il soit clair que c'est la console web et pas le terminal qui a lancé sensei.

Reply:
"browser console"

Reply:
![image](https://user-images.githubusercontent.com/1656170/122895895-d041a080-d348-11eb-82d0-0756a4f85cf7.png)

Reply:
I changed the warning for an error.